filesystem/apps/terminal/window.rs

```diff
@@ -19,20 +18,20 @@ pub struct ConsoleWindow {
     /// Previous commands
     pub history: Vec<String>,
     /// History index
-    pub history_i: usize,
+    pub history_i: u32,
     /// Offset
-    pub offset: usize,
+    pub offset: u32,
```

Comment:
Again, I don't want to see offset be `u32` because of the number of awkward, unnecessary conversions